packages/sw-lib/src/lib/sw-lib.js

```diff
@@ -351,6 +352,25 @@ class SWLib {
       requestWrapper: new RequestWrapper(wrapperOptions),
     });
   }
+
+  /**
+   * This method will register any default routes the library will need.
+   */
+  _registerDefaultRoutes() {
+    const revisionedManager = this._precacheManager.getRevisionedCacheManager();
+    const cacheFirstHandler = this.cacheFirst({
+      cacheName: revisionedManager.getCacheName(),
+    });
+
+    const route = new this.Route({
+      match: ({url, event}) => {
+        const cachedUrls = revisionedManager.getCachedUrls();
```

Comment:
I can't call it outside of the function as the precache entries won't be defined until *after* this route is defined.

What I could do however is make the function call trivial by generating the array in the install event when the list of URL's is "finalised" / can't change. What that be ok with you?